src/solver/conv_mlir_igemm_fwd.cpp

```diff
@@ -37,6 +37,14 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_MLIR_IGEMM_FWD)
 namespace miopen {
 namespace solver {
 
+const PerformanceConvMlirIgemm& PerformanceConvMlirIgemm::GetHeuristicInitRequest()
+{
+    // Special value to represent the heuristic initialized perf_config.
```

Comment:
I think that this value represents the request to MLIR to create the default PerformanceConfig (possibly heuristically initialized for the best performance). I recommend either removing the comment or making it crystal clear ;) Perhaps the best solution is:
```C++
/// \ref https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1154
```
Or
```
/// See issue 1154
```

I also highly recommend moving this comment to the header, near the declaration of `PerformanceConvMlirIgemm::GetHeuristicInitRequest()`.